azure-pipelines-pr-validation.yml

```diff
@@ -338,6 +338,7 @@ extends:
         publishingInfraVersion: 3
         # Symbol validation is not entirely reliable as of yet, so should be turned off until
         # https://github.com/dotnet/arcade/issues/2871 is resolved.
+        enableSigningValidation: false
```

Comment:
The comment above probably does not apply to this line.

Reply:
Thanks!